cos_alerter/server.py

```diff
@@ -5,16 +5,39 @@
 
 import logging
 
-from flask import Flask, request
+import timeago
+from flask import Flask, render_template, request
 from prometheus_flask_exporter import PrometheusMetrics
 
-from .alerter import AlerterState, config
+from .alerter import AlerterState, config, now_datetime
 
 app = Flask(__name__)
 metrics = PrometheusMetrics(app)
 logger = logging.getLogger(__name__)
 
 
+@app.route("/", methods=["GET"])
+def dashboard():
+    """Endpoint for the COS Alerter dashboard."""
+    clients = []
+    now = now_datetime()
+    for clientid in AlerterState.clients():
+        with AlerterState(clientid) as state:
+            last_alert = state.last_alert_datetime()
+            alert_time = timeago.format(last_alert, now) if last_alert is not None else "never"
+            status = "✅ Up" if not state.is_down() else "❌ Down"
```

Comment:
[recommendation] it's been some time since my last jinja, but:
- I'd rename dashboard.html to dashboard.jinja to make it clear it's a template
- I'd expect the emoji's and 'textual' parts of the rendering to be handled by the template. In other words, instead of injecting "✅ Up" in the template, we should inject a boolean (True for example) and let the template decide how to render that. This way it's easier to keep the template and the code passing values to the template in sync should there ever be changes.

Reply:
* I think the is the standard naming convention for flask apps. It is in a directory called templates after all.
* Done.